src/web_app/html/params.html

```diff
@@ -89,8 +89,5 @@
     <a href="//github.com/GoogleChrome/webrtc" title="View the repository" id="viewSource">github.com/GoogleChrome/webrtc</a>
 
   </div>
-
```

Comment:
@jiayliu 
Was this intentional?
Why is it missing from the pull request description?